docs/api.rst

```diff
@@ -5892,8 +5892,7 @@ Export PnL report debug data
                 "evmchains_to_skip_detection": []
             },
             "ignored_events_ids": {
-                "trade": ["X124-JYI", "2325"],
-                "ethereum transaction": ["0xfoo", "0xboo"]
+                "history_event": ["0xfoo", "0xboo"]
```

Comment:
you did not remove this .... as I said (https://github.com/rotki/rotki/pull/9809/files#r2051059652) the intermediate history event key should go away